src/components/DropzoneAreaFunc.js

```diff
@@ -0,0 +1,486 @@
+  previewChipProps: PropTypes.object,
+  /**
+   * Custom CSS classNames for preview Grid components.<br/>
+   * Should be in the form {container: string, item: string, image: string}.
+   */
+  previewGridClasses: PropTypes.object,
+  /**
+   * Props to pass to the Material-UI Grid components.<br/>
+   * Should be in the form {container: GridProps, item: GridProps}.
+   *
+   * @see See [Material-UI Grid](https://material-ui.com/api/grid/#props) for available GridProps values.
+   */
+  previewGridProps: PropTypes.object,
+  /** The label for the file preview section. */
+  previewText: PropTypes.string,
+  /** Shows styled Material-UI Snackbar when files are dropped, deleted or rejected. */
+  showAlerts: PropTypes.bool,
+  /** Clear uploaded files when component is unmounted. */
+  clearOnUnmount: PropTypes.bool,
+  /** List of URLs of already uploaded images.<br/>**Note:** Please take care of CORS. */
+  initialFiles: PropTypes.arrayOf(PropTypes.string),
+  /**
+   * Get alert message to display when files limit is exceed.
+   *
+   * *Default*: "Maximum allowed number of files exceeded. Only ${filesLimit} allowed"
+   *
+   * @param {number} filesLimit The `filesLimit` currently set for the component.
+   */
+  getFileLimitExceedMessage: PropTypes.func,
+  /**
+   * Get alert message to display when a new file is added.
+   *
+   * *Default*: "File ${fileName} successfully added."
+   *
+   * @param {string} fileName The newly added file name.
+   */
+  getFileAddedMessage: PropTypes.func,
+  /**
+   * Get alert message to display when a file is removed.
+   *
+   * *Default*: "File ${fileName} removed."
+   *
+   * @param {string} fileName The name of the removed file.
+   */
+  getFileRemovedMessage: PropTypes.func,
+  /**
+   * Get alert message to display when a file is rejected onDrop.
+   *
+   * *Default*: "File ${rejectedFile.name} was rejected."
+   *
+   * @param {Object} rejectedFile The file that got rejected
+   * @param {string[]} acceptedFiles The `acceptedFiles` prop currently set for the component
+   * @param {number} maxFileSize The `maxFileSize` prop currently set for the component
+   */
+  getDropRejectMessage: PropTypes.func,
+  /**
+   * Fired when the files inside dropzone change.
+   *
+   * @param {File[]} loadedFiles All the files currently loaded into the dropzone.
+   */
+  onChange: PropTypes.func,
+  /**
+   * Fired when the user drops files into the dropzone.
+   *
+   * @param {File[]} droppedFiles All the files dropped into the dropzone.
+   */
+  onDrop: PropTypes.func,
+  /**
+   * Fired when a file is rejected because of wrong file type, size or goes beyond the filesLimit.
+   *
+   * @param {File[]} rejectedFiles All the rejected files.
+   */
+  onDropRejected: PropTypes.func,
+  /**
+   * Fired when a file is deleted from the previews panel.
+   *
+   * @param {File} deletedFile The file that was removed.
+   */
+  onDelete: PropTypes.func,
+};
+
+export default withStyles(styles)(DropzoneAreaFunc);
```

Comment:
`withStyles` should be migrated to `makeStyles` from `Material-UI`, see [here](https://material-ui.com/styles/basics/#getting-started) for more details.

Reply:
+1